spec/date.html

```diff
@@ -477,6 +477,9 @@ <h1>Abstract operations</h1>
       <h1>CreateTemporalDate ( _isoYear_, _isoMonth_, _isoDay_ [ , _newTarget_ ] )</h1>
       <emu-alg>
         1. Perform ? RejectDate(_isoYear_, _isoMonth_, _isoDay_).
+        1. NOTE: Deferring to ValidateDateTimeRange with an hour of 12 avoids trouble at the extremes of the representable range of dates, which stop just before midnight on each end.
```

Comment:
I suppose this could be an `emu-note[type=editorial]` down below instead?

Reply:
TIL

Reply:
I left out `type="editor"` since the ecmarkup docs say that the blank type is intended for implementers, which I think is the category that this note belongs to.